src/js/base/ripe.js

```diff
@@ -811,6 +811,7 @@ ripe.Ripe.prototype.bindConfigurator = function(element, options = {}) {
  */
 ripe.Ripe.prototype.bindInteractable = function(element) {
     this.children.push(element);
+    this.trigger("new_child", element);

```

Comment:
@joamag do you think this makes sense? Maybe this or `interactable_bound`.

We were considering using this in order to be able to bind to the events of children, if they were added in runtime.

Reply:
let's just call them "bound" and "unbound"